lib/generators/super_good/solidus_taxjar/install/install_generator.rb

```diff
@@ -12,6 +12,10 @@ def create_initializer_file
             Spree.config do |config|
               config.tax_calculator_class = SuperGood::SolidusTaxjar::TaxCalculator
             end
+            if Spree.solidus_gem_version < Gem::Version.new('2.11.0')
```

Comment:
In the spirit of Ben's earlier comment of not putting code that has the power to affect how the reporting feature works in the initializer, we should move this to internal gem code.

One place could be inside the body of the rails engine

````rb
# frozen_string_literal: true

module SuperGoodSolidusTaxjar
  class Engine < Rails::Engine
    isolate_namespace Spree
    engine_name 'super_good_solidus_taxjar'

    if Spree.solidus_gem_version < Gem::Version.new('2.11.0')
      require root.join('app/subscribers/super_good/solidus_taxjar/spree/reporting_subscriber')
      SuperGood::SolidusTaxjar::Spree::ReportingSubscriber.subscribe!
    end

    include SolidusSupport::EngineExtensions
  end
end
````